frontend/src/toolbar/elements/heatmapLogic.ts

```diff
@@ -53,7 +53,10 @@ export const heatmapLogic = kea<heatmapLogicType<ElementsEventType, CountedHTMLE
                         properties: [{ key: '$current_url', value: $current_url }],
                         temporary_token: toolbarLogic.values.temporaryToken,
                     }
-                    const url = `${toolbarLogic.values.apiURL}api/element/stats/${encodeParams(params, '?')}`
+                    const url = `${toolbarLogic.values.apiURL}api/projects/@current/elements/stats/${encodeParams(
```

Comment:
Is `projects/@current` correct for toolbar or should this come from the project the toolbar is active for :thinking:

Reply:
Hmm, right. Well, it seems then that the toolbar needs additional logic to handle multiple projects as well. 🤔 @mariusandra?

Reply:
So the thing is this just doesn't change anything for the toolbar, because it isn't and hasn't been adjusted for multiple project yet anyway. But definitely, the toolbar should be project ID-dependent.